tests/main.nf.test

```diff
@@ -593,10 +593,18 @@ nextflow_workflow {
                             [2,1,'Na/K ATPase',[],[],[],100,[],[]],
                             [3,1,'CD3',[],[],[],100,[],[]],
                             [4,1,'CD45RO',[],[],[],100,[],[]],
-                            [5,2,'DNA 2',[],[],[],100,[],'TRUE'],
-                            [6,2,'Antigen Ki67',[],[],[],100,'Na/K ATPase',[]],
-                            [7,2,'Pan-cytokeratin',[],[],[],100,'CD3',[]],
-                            [8,2,'Aortic smooth muscle actin',[],[],[],100,'CD45RO',[]]
+                            [5,1,'DNA 2',[],[],[],100,[],[]],
```

Comment:
Why are there double the number of entries (with duplications) in the markersheet input channel now?

Reply:
I had put the wrong input file in for this test and it was causing backsub to need additional rows.
Thanks for discussing this and helping me to find where I went wrong @jmuhlich!